src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Circle extends Figure {
+    private String name = "circle";
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
```

Comment:
Remove random logic from the Circle class.
It should contain only its own properties(radius, color) and implement `getArea` and `draw` methods.